spec/source/conventions.html

```diff
@@ -0,0 +1,107 @@
+<emu-clause id="conventions">
+  <h1>Notational Conventions</h1>
+
+  <p>
+    This standard uses a subset of the notational conventions of the ECMAScript 2015 Language Specification (ECMA-262 6th Edition), as ES2015:
+  </p>
+
+  <ul>
+    <li>Object Internal Methods and Internal Slots, as described in ES2015, 6.1.7.2.</li>
```

Comment:
@rwaldron we might need a way to link to ES2015 for all those sections that we reference here. I wonder if we have a resource list that can be generated but 262 that we can import to enable all those links :), or maybe that's just too much. /cc @bterlson